courses/catalog_serializers.py

```diff
@@ -71,12 +76,66 @@ def get_thumbnail_url(self, program):
         except ProgramPage.DoesNotExist:
             return None
 
+    def get_instructors(self, program):
+        """Get the list of instructors from the program page"""
+        try:
+            page = program.programpage
+        except ProgramPage.DoesNotExist:
+            return []
+
+        return list(page.faculty_members.values("name"))
+
+    def get_total_price(self, program):
+        """Get the combined price of all courses"""
+        return str(program.price * program.num_required_courses)
```

Comment:
Can this be a numeric field instead, as in xpro?

Reply:
The result of `program.price * program.num_required_courses` is a `Decimal`, so if we render it as a string here and then parse as a `Decimal` in javascript, it removes the risk of floating point rounding errors

Reply:
👍